pennylane/templates/subroutines/qrom.py

```diff
@@ -198,13 +198,18 @@ def decomposition(self):  # pylint: disable=arguments-differ
     def compute_decomposition(
         bitstrings, control_wires, target_wires, work_wires, clean
     ):  # pylint: disable=arguments-differ
+
+        if len(control_wires) == 0:
+            return [qml.BasisEmbedding(int(bits, 2), wires=target_wires) for bits in bitstrings]
```

Comment:
This is added to cover the case of QROM where there are no control_wires, i.e., we only want to load a bitstring